src/Microsoft.Extensions.Logging.Abstractions/LogLevel.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace Microsoft.Extensions.Logging
 {
+    /// <summary>
+    /// Defines logging severity levels.
+    /// </summary>
     public enum LogLevel
     {
         Debug = 1,
```

Comment:
We can perhaps copy some of the guidance from https://github.com/aspnet/Logging/wiki/Guidelines to the enum values.